docs/usage/configuration-options.md

```diff
@@ -3558,6 +3558,13 @@ Configure this to `true` if you wish to get one PR for every separate major vers
 e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
 If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.
 
+## separateMultipleMinor
+
+Enable this for dependencies when it's important to split updates into separate PRs per minor release stream (e.g. `python`).
+
+e.g. if you are on python@v3.9.0 currently then default behavior is a PR for upgrading to `python@v3.11.x` and not for `python@v3.10.x`.
+If this setting is true then you would get separate PRs for `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`.
```

Comment:
```suggestion
If this setting is true then you would get separate PRs for each minor stream, such as `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`, etc..
```

Reply:
Nitpick: I don't believe you need to add a period after abbreviations with periods, so just etc. 
```suggestion
If this setting is true then you would get separate PRs for each minor stream, such as `python@3.9.x`, `python@v3.10.x`, `python@v3.11.x`, etc.
```